src/wfpt.pyx

```diff
@@ -70,6 +75,153 @@ def wiener_like(np.ndarray[double, ndim=1] x, double v, double sv, double a, dou
     return sum_logp
 
 
+def wiener_like_rlddm(np.ndarray[double, ndim=1] x,
+                      np.ndarray[long, ndim=1] response,
+                      np.ndarray[double, ndim=1] feedback,
+                      np.ndarray[long, ndim=1] split_by,
+                      double q, double alpha, double dual_alpha, double v, 
+                      double sv, double a, double z, double sz, double t,
+                      double st, double err, int n_st=10, int n_sz=10, bint use_adaptive=1, double simps_err=1e-8,
+                      double p_outlier=0, double w_outlier=0):
+    cdef Py_ssize_t size = x.shape[0]
+    cdef Py_ssize_t i
+    cdef Py_ssize_t s_size
+    cdef int s
+    cdef double p
+    cdef double sum_logp = 0
+    cdef double wp_outlier = w_outlier * p_outlier
+    cdef double alfa
+    cdef np.ndarray[double, ndim=1] qs = np.array([q, q])
```

Comment:
And it's correct that you are just initializing with `q, q` once at the beginning? I.e. we didn't introduce a bug by moving it up here?

Reply:
Moving it up does not change things for how I've set it up now. In the future it could be that people would want to have separate initial q-values for upper and lower bound alternatives and/or across split_by (conditions).